src/Ast/AST.php

```diff
@@ -73,9 +117,9 @@ public function toCode(): string
 
     /**
      * Create a PHP variable.
-     * 
```

Comment:
It might be worth moving all the whitespace changes into a separate commit, if you have the energy.

Reply:
I agree this would be better in a separate commit or PR, but think I'll leave it as-is this time.